Watchman.Discord/Areas/Messaging/BotCommands/SendCommand.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Devscord.DiscordFramework.Framework.Commands;
+using Devscord.DiscordFramework.Framework.Commands.PropertyAttributes;
+
+namespace Watchman.Discord.Areas.Messaging.BotCommands
+{
+    public class SendCommand : IBotCommand
+    {
+        [ChannelMention]
```

Comment:
Dobrze byłoby dodać konwersje z `ChannelMention` do `ulong` 

Tak jak mamy konwersje z przykładowo 
`Number` do `int`
`Time` do `TimeStamp` 
itd

I to samo dla `UserMention`

Wtedy dostawalibyśmy od razu ID, przez co obsługa byłaby dużo prostsza